lotus/metrics.go

```diff
@@ -13,4 +19,123 @@ var (
 		Description: "Current height of Lotus node",
 		Aggregation: view.LastValue(),
 	}
+
+	metricHeightInterval = time.Second * 120
 )
+
+// SyncMonitor provides information about the Lotus
```

Comment:
TL;DR: SyncMonitor analyzes the sync status every 2 minutes, and provides cached value to whoever wants to know about it.